app/controllers/admins_controller.rb

```diff
@@ -43,7 +43,7 @@ def weekly_user_stats
     @created_users = User.where("username IS NOT NULL and created_at IS NOT NULL")
     @created_users.find_each do |u|
       week = u.created_at.beginning_of_week.strftime("%Y-%m-%d")
-      @created_users_by_week[week] << u.username
+      @created_users_by_week[week] << {"username" => u.username, "closed_account" => u.person.closed_account}
```

Comment:
Two things here:

1. In Ruby, there is a concept called "symbols", which are basically strings, except for not being real strings. They are much lighter in their memory consumption. In general, for Array/Object keys, instead of `"username"`, it's much better to use `:username`.
2. Using strings also enables you to use a bit of syntax sugar here. So instead of using quotes and arrows, you can re-write this as `{username: u.username, closed_account: u.person.closed_account}`, which is a lot nicer on the eyes!

When accessing the array value, you can replace `m["username"]` with `m[:username]` after transforming this to symbols.

Reply:
Thank you very much for your comments. I replaced strings with symbols and changed the syntax.

All this is very mew to me.